1.2 Inventory.feature

```diff
@@ -0,0 +1,43 @@
+Feature: 1.2 Inventory
+
+Background: 
+Given main page URL is opened
+
+@Smoke
+Scenario: 1.2_01 Main scenario: user should be able to add items in the cart to buy them
+	Given user is on the main page
+	And user click on "Add to cart" button
+	When Item appears in the cart
```

Comment:
These are not high-level tests, here we have to write specific data - where they clicked, what they added, so that later we can check whether it was added accurately and, if necessary, delete and check the deletion. If there is no specific data, we will not be able to automate this test. And then why do we need such tests?